src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/multiarch.py

```diff
@@ -166,35 +157,30 @@ def create_build(location,
         except FileNotFoundError as e:
             logger.warning("Unable to find file %s", str(e))
 
-def acr_multi_build_definition_create(registry_name,
-                                      repository,
-                                      tags=None,
+def acr_multi_build_definition_create(registry,
```

Comment:
Revert `registry` back to `registry_name` and register it similar to https://github.com/AppPlatPerf/azure-cli/blob/master/src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py#L49